src/ext/java/org/opentripplanner/ext/vectortiles/layers/stops/DigitransitRealtimeStopPropertyMapper.java

```diff
@@ -0,0 +1,64 @@
+package org.opentripplanner.ext.vectortiles.layers.stops;
+
+import static org.opentripplanner.ext.vectortiles.layers.stops.DigitransitStopPropertyMapper.getRoutes;
+import static org.opentripplanner.ext.vectortiles.layers.stops.DigitransitStopPropertyMapper.getType;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.opentripplanner.apis.support.mapping.PropertyMapper;
+import org.opentripplanner.framework.i18n.I18NStringMapper;
+import org.opentripplanner.inspector.vector.KeyValue;
+import org.opentripplanner.transit.model.site.RegularStop;
+import org.opentripplanner.transit.service.TransitService;
+
+public class DigitransitRealtimeStopPropertyMapper extends PropertyMapper<RegularStop> {
+
+  private final TransitService transitService;
+  private final I18NStringMapper i18NStringMapper;
+
+  public DigitransitRealtimeStopPropertyMapper(TransitService transitService, Locale locale) {
+    this.transitService = transitService;
+    this.i18NStringMapper = new I18NStringMapper(locale);
+  }
+
+  protected static DigitransitRealtimeStopPropertyMapper create(
+    TransitService transitService,
+    Locale locale
+  ) {
+    return new DigitransitRealtimeStopPropertyMapper(transitService, locale);
+  }
+
+  @Override
+  protected Collection<KeyValue> map(RegularStop stop) {
+    String alerts = JSONArray.toJSONString(
+      transitService
+        .getTransitAlertService()
+        .getStopAlerts(stop.getId())
+        .stream()
+        .map(alert -> {
+          JSONObject alertObject = new JSONObject();
+          alertObject.put("alertEffect", alert.effect().toString());
+          return alertObject;
+        })
```

Comment:
What do we actually need from the alerts? So far (as far as I know), we haven't utilized the alert effect for anything in our UI. Is this so data producers can send NO_SERVICE affect when stop is  closed?

Reply:
I think the alert validity period would at least be something we would care about as we don't want to display stop as closed if the alert is for future/past.

Reply:
If we are only interested in the NO_EFFECT alerts and nothing else, alternative solution could be to just have some `isClosed` field instead of exposing the alerts. The logic could be just are there any NO_SERVICE alerts that are currently active.

Reply:
I'm not really sure what is the best either. Exposing objects/lists through the vector tiles is a bit ugly (I know we already do it) because you have to encode them but on the other hand, `isClosed` would be a slightly cryptic field as it's not immediately clear where that information comes from. Does @leonardehrenfried have some opinion about this?

Reply:
I find the lists uglier than a potentially confusing `isClosed`. To be less confusing, you could add use enum values like `state=OPERATIONAL`, `state=OUT_OF_SERVICE`, `state=CLOSED`, `state=OUT_OF_SERVICE`.

Reply:
Not sure if I'm the biggest fan of enums in a map layer as we would need to document somewhere the potential values (same issue with the alert effect types but at least they are somewhat standard).

Reply:
And I just realised it could be read as "there are no service alerts", back to drawing board then...